### PR TITLE
Ncnews 8

### PR DESCRIPTION
**Pull request by ckloudy**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -7,131 +7,162 @@ const testData = require('../db/data/test-data/index');
...
+						expect(article).toHaveProperty('votes');
+						expect(article).toHaveProperty('comment_count');
+						expect(article.author).toEqual(expect.any(String));
+						expect(article.title).toEqual(expect.any(String));
+						expect(article.article_id).toEqual(expect.any(Number));
+						expect(article.topic).toEqual(expect.any(String));
+						expect(article.created_at).toEqual(expect.any(String));
+						expect(article.votes).toEqual(expect.any(Number));
+						expect(article.comment_count).toEqual(expect.any(Number));
+					});
```
Comment (scar1377) [inline __tests__/app.test.js:42]: better to write a new test for ticket 8 and only test comment_count property

File: models/articles.model.js [hunk 2]
```diff
@@ -1,13 +1,27 @@
 const db = require('../db/connection');
 
+exports.selectArticles = () => {
+	return db
+		.query(
+			`SELECT articles.author, articles.title, articles.article_id, articles.body, articles.topic, articles.created_at, articles.votes, CAST(COUNT(comments) AS INT) AS comment_count
+			FROM articles
```
Comment (scar1377) [inline models/articles.model.js:7]: don't need articles.body for this one

**Review by scar1377 — APPROVED** [review]
don't need body, also better to write a new test. Can go ahead make the change and merge it

File: __tests__/app.test.js (hunk 1 above)
Comment (ckloudy) [reply __tests__/app.test.js:42]: This is a new test for the /api/articles end point. It was checking the article appears as it should. I realise that a previous test was checking some of this so have removed all but the comment_count property.

**Pull request merged**